### ISSUE TITLE
Remove old transforming format

### ISSUE THREAD

**Issue by jessebrennan** [jessebrennan/newt-transformer#30]
This is no longer used. It is referenced in a couple places and should be cleaned up. This will probably necessitate a new release.

┆Issue is synchronized with this [JIRA Story](https://ucsc-cgl.atlassian.net/browse/NEWT-26)
┆Project Name: CGP-Newt
┆Issue Number: NEWT-26

**Comment by cricketsloan** [comment]
[~brennan] Do you want to clean this up before we have newt iced for awhile?

**Comment by jessebrennan** [comment]
Good idea. I won't prioritize it, but I'll take care of it while I have time.
